src/screens/settings-screen/settings-screen.tsx

```diff
@@ -109,6 +124,39 @@ export const SettingsRootScreen: FC<
                 onValueChange={setTheme}
               />
             )}
+            <SettingsUI.SwitchItem
+              label="Custom rotating circle color"
+              secondaryLabel="Change rotating circle color"
+              iconName="ios-moon"
+              iconBackgroundColor="#a5b4fc"
+              value={customRotatingCircleColor}
+              onValueChange={setCustomRotatingCircleColor}
+            />
+            {customRotatingCircleColor && (
+              <View style={{ flexDirection: "column" }}>
+                <View
+                  style={{
+                    width: 25,
+                    height: 25,
+                    backgroundColor: colors.pastel[rotatingCircleColor],
+                    marginLeft: 23,
+                  }}
```

Comment:
Can we follow the same pattern used in the other settings and create a separate android and ios entry in https://github.com/mmazzarolo/breathly-app/blob/master/src/screens/settings-screen/settings-ui.ios.tsx and https://github.com/mmazzarolo/breathly-app/blob/master/src/screens/settings-screen/settings-ui.android.tsx?

Reply:
I ran into some issues with the circle color preview, so I ended up removing it in my recent changes.